types/reactstrap/reactstrap-tests.tsx

```diff
@@ -3261,7 +3261,7 @@ class Example98 extends React.Component {
 class Example99 extends React.Component {
   render() {
     return (
-      <Table responsive>
+      <Table responsive="md">
```

Comment:
Can you add this as a separate test?  We want to test `responsive` as both boolean and string.

Reply:
Scratch that, just noticed that Example100 is exactly the same as 99